### PR TITLE
docs: minor updates to the quickstart

### PR DESCRIPTION
**Pull request by Thab310**
Minor updates to the "Quickstart" page in terms of grammar and spelling.

Signed-off-by: Thabelo Ramabulana <thabelo.dev@gmail.com>

### CODE REVIEW

File: docs/src/quickstart.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Quickstart
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
 [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
 
-!!! Warning
+> [!WARNING]
```
Comment (jsilvela) [inline docs/src/quickstart.md:6]: These changes are not compatible with the MkDocs format for admonitions.
See https://squidfunk.github.io/mkdocs-material/reference/admonitions/

File: docs/src/quickstart.md [hunk 2]
```diff
@@ -232,7 +231,7 @@ prometheus-community-kube-operator        ClusterIP   443/TCP
 prometheus-community-kube-prometheus      ClusterIP   9090/TCP
 ```
 
-### Viewing with Prometheus
+### Monitoring with Prometheus
```
Comment (jsilvela) [inline docs/src/quickstart.md:234]: I don't see an increase in clarity with this change.
Generally "monitoring" is used to mean an ongoing periodic activity.
Not 100% against, but I also don't see strong reason for it.

File: docs/src/quickstart.md [hunk 3]
```diff
@@ -307,12 +306,12 @@ CloudNativePG provides a default dashboard for Grafana as part of the official
...
+> [!WARNING]
     Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time
     this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
     producing some graphs to show `No data`.
 
 ![local grafana](images/grafana-local.png)
 
-Note that in our example setup, both Prometheus and Grafana will pick up
-any other CloudNativePG clusters deployed with Monitoring activated.
+> [!NOTE]
```
Comment (jsilvela) [inline docs/src/quickstart.md:316]: If you mean to have this as a boxed aside, please use the !!! admonition format.
But ... is this worth and admonition box?

**Review by jsilvela — CHANGES_REQUESTED** [review]
Thanks for your contribution @Thab310 ,
the changes you are proposing to the admonition format won't work.
Please check the README.md file in the `/docs` folder to see how to
serve the files with MkDocs.

File: docs/src/quickstart.md (hunk 1 above)
Comment (Thab310) [reply docs/src/quickstart.md:6]: Thanks for your feedback. I appreciate it.🙏

File: docs/src/quickstart.md (hunk 2 above)
Comment (Thab310) [reply docs/src/quickstart.md:234]: I understand @jsilvela.
I would love to contribute to cnpg, are there any tips you would give someone who is a beginner to open-source contributions and cnpg to be more precise?

File: docs/src/quickstart.md (hunk 3 above)
Comment (Thab310) [reply docs/src/quickstart.md:316]: That was my error i didn't know that it's not compatible.

File: docs/src/quickstart.md (hunk 2 above)
Comment (jsilvela) [reply docs/src/quickstart.md:234]: Trying to improve the documentation, as you are trying to do, is a good way @Thab310 , and just to be clear, I haven't REJECTED your initiative but asked you to make changes.
If you're looking for other things to do, you might want to try with the "good first issue" items.
https://github.com/cloudnative-pg/cloudnative-pg/labels/good%20first%20issue

File: docs/src/quickstart.md (hunk 3 above)
Comment (Thab310) [reply docs/src/quickstart.md]: @jsilvela I actually did not know about admonition format. I did some research and managed to update the requested changes and created a new commit. 🙏Please let me know if there is anything else you'd like for me to update.

**Review by Thab310 — COMMENTED** [review]
@jsilvela Thanks for your feedback, I managed to update the pull request proposed changes.🙏

File: docs/src/quickstart.md [hunk 4]
```diff
@@ -1,23 +1,17 @@
...
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
 [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
 
-!!! Warning
-    The instructions contained in this section are for demonstration,
-    testing, and practice purposes only and must not be used in production.
+!!! Warning The instructions contained in this section are for demonstration, testing, and practice purposes only and must not be used in production.
```
Comment (jsilvela) [inline docs/src/quickstart.md:6]: Sorry. No. Why are you changing the spacing and layout?
Again see https://squidfunk.github.io/mkdocs-material/reference/admonitions/
for the layout of admonition boxes.

File: docs/src/quickstart.md [hunk 5]
```diff
@@ -1,23 +1,17 @@
...
 regular manifests written in YAML.
 
 By following the instructions on this page you should be able to start a PostgreSQL
 cluster on your local Kubernetes installation and experiment with it.
 
-!!! Important
-    Make sure that you have `kubectl` installed on your machine in order
-    to connect to the Kubernetes cluster. Please follow the Kubernetes documentation
-    on [how to install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+!!! Important Make sure that you have `kubectl` installed on your machine in order to connect to the Kubernetes cluster. Please follow the Kubernetes documentation on [how to install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
```
Comment (jsilvela) [inline docs/src/quickstart.md:14]: Please undo this and all the changes that alter the layout of admonition boxes.
They're the way they are for a reason.

File: docs/src/quickstart.md [hunk 6]
```diff
@@ -93,9 +87,7 @@ spec:
     size: 1Gi
 ```
 
-!!! Note "There's more"
-    For more detailed information about the available options, please refer
-    to the ["API Reference" section](cloudnative-pg.v1.md).
+!!! Note For more detailed information about the available options, please refer to the ["API Reference" section](cloudnative-pg.v1.md).
```
Comment (jsilvela) [inline docs/src/quickstart.md:90]: Why did you take out the title of the note?

**Review by jsilvela — CHANGES_REQUESTED** [review]
Please note that the documentation is written with 80 column-wide lines.
Please respect that in your changes.
I think there are very few actual wording changes that you're proposing.
The diff should correspond to that.

**Review by Thab310 — COMMENTED** [review]
I appreciate your patience @jsilvela

**Review by jsilvela — CHANGES_REQUESTED** [review]
Again, @Thab310 , there are way too many hits in the diff relative the the actual wording changes you're proposing.
Please look at the diff view in GitHub, or locally `git diff main...yourbranch` and ensure only your real wording changes appear.

A few examples in the current push:

1. Adding a trailing space to the lines (please always avoid, your local `git diff` should also be showing these clearly).
Screenshot 2024-06-07 at 11 37 41

2. Again, messing with admonition boxes. `Seealso` is there for a reason and is a known element of the admonition format.
Screenshot 2024-06-07 at 11 37 27

**Review by Thab310 — COMMENTED** [review]
@jsilvela I removed all the trailing spaces. How does it look now?

File: docs/src/quickstart.md [hunk 7]
```diff
@@ -1,7 +1,6 @@
 # Quickstart
 
-This section describes how to test a PostgreSQL cluster on your laptop/computer
-using CloudNativePG on a local Kubernetes cluster in [Kind](https://kind.sigs.k8s.io/) or
+This section guides you through testing a PostgreSQL cluster on your local machine by deploying CloudNativePG on a local Kubernetes cluster using either [Kind](https://kind.sigs.k8s.io/) or
```
Comment (gbartolini) [inline docs/src/quickstart.md:3]: We normally break a line around 75-80 chars.

**Review by Thab310 — COMMENTED** [review]
@gbartolini thanks for your feedback. How does it look now?

File: docs/src/quickstart.md [hunk 8]
```diff
@@ -308,11 +309,11 @@ CloudNativePG provides a default dashboard for Grafana as part of the official
 file and manually importing it via the GUI.
 
 !!! Warning
-    Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time
-    this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
+    Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time 
+    this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
```
Comment (gbartolini) [inline docs/src/quickstart.md:313]:
```suggestion
    Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time
    this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
```

File: docs/src/quickstart.md [hunk 9]
```diff
@@ -308,11 +309,11 @@ CloudNativePG provides a default dashboard for Grafana as part of the official
...
-    this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes`
+    Some graphs in the previous dashboard make use of metrics that are in alpha stage by the time 
+    this was created, like `kubelet_volume_stats_available_bytes` and `kubelet_volume_stats_capacity_bytes` 
     producing some graphs to show `No data`.
 
 ![local grafana](images/grafana-local.png)
 
-Note that in our example setup, both Prometheus and Grafana will pick up
-any other CloudNativePG clusters deployed with Monitoring activated.
+Note that in our local setup, Prometheus and Grafana are configured to automatically discover
```
Comment (gbartolini) [inline docs/src/quickstart.md:318]:
```suggestion
Note that in our local setup, Prometheus and Grafana are configured to automatically discover
```

**Review by gbartolini — APPROVED** [review]

**Pull request merged**